olive/platform_sdk/qualcomm/snpe/tools/dev.py

```diff
@@ -38,8 +38,9 @@ def get_dlc_io_config(dlc_path: str, input_names: List[str], output_names: List[
     input_names: list of input names of source model.
     output_names: list of output names of source model.
     """
-    tmp_csv = tempfile.NamedTemporaryFile(suffix=".csv")  # pylint: disable=consider-using-with
-    dlc_info = get_dlc_info(dlc_path, csv_path=tmp_csv.name)
+    tmp_folder = tempfile.TemporaryDirectory()
```

Comment:
Will meet permission issue when create a temp file outside the sub process but the the file is needed in sub process